libexec/bats

```diff
@@ -55,7 +60,7 @@ expand_path() {
 BATS_LIBEXEC="$(abs_dirname "$0")"
 export BATS_PREFIX="$(abs_dirname "$BATS_LIBEXEC")"
 export BATS_CWD="$(abs_dirname .)"
-export PATH="$BATS_LIBEXEC:$PATH"
+export PATH="$BATS_LIBEXEC:$(getconf PATH 2>/dev/null):$PATH"
```

Comment:
So I just tested my Solaris (10 U11) test suite without the getconf change, and bats works fine

So this change implies it is for the *tests* to use POSIX binaries, and not bats itself. I too would leave that as an exercise for the tester to set the correct environment for the tests, as we would be testing with the SV binaries